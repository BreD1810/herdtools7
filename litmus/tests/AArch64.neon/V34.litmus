AArch64 V34
(* Test Neon final state checking
   LD1 single structure *)
{
x = 22;
0:X0=x;
}
 P0                     ;
 LD1 {V0.S}[0],[X0]     ;
forall(0:V0.S = {22,0,0,0})
