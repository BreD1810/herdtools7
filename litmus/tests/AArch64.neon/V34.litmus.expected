Test V34 Required
Histogram (1 states)
1000000:>0:V0.S={22,0,0,0};
Ok

Witnesses
Positive: 1000000, Negative: 0
Condition forall (0:V0.S={22,0,0,0}) is validated
Hash=6adf8227cefdaf54aa8b354007fb3bd3
Observation V34 Always 1000000 0
